stripe/src/main/java/com/stripe/android/paymentsheet/analytics/SessionId.kt

```diff
@@ -0,0 +1,14 @@
+package com.stripe.android.paymentsheet.analytics
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.util.UUID
+
+@Parcelize
+internal class SessionId private constructor(
```

Comment:
Might be worth making this a data class

Reply:
I was considering doing this, but then you effectively can't have a private constructor because it exposes the `copy` method.